model/project.go

```diff
@@ -387,17 +393,16 @@ func (tt TaskIdTable) GetIdsForAllTasks(currentVariant, taskName string) []strin
 }
 
 // TaskIdTable builds a TaskIdTable for the given version and project
-func NewTaskIdTable(p *Project, v *version.Version) TaskIdTable {
+func NewTaskIdTable(p *Project, v *version.Version) (TaskIdTable, TaskIdTable) {
```

Comment:
I'd rather have this return a struct with two fields one for display and one for execution that would make it less likely that someone was going to get the order backwards.

Reply:
done